src/css/profile/mobile/common/button.less

```diff
@@ -299,6 +299,10 @@ a.ui-btn {
 			&-pause::after {
 				mask-image: url("images/controls/00_button_pause.png");
```

Comment:
It is recommended to fit the format. How about removing double quotes?

Reply:
I simple forget about quotes... most of css in TAU has quotes. If there was a rule then we should follow it.